.circleci/config.yml

```diff
@@ -33,11 +27,21 @@ jobs:
           paths:
             - vendor/
             - /usr/local/Cellar
-            
+
+jobs:
+  test:
+    executor:
+      name: ios/default
+      xcode-version: "11.2.1"
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: 1
+      PKG_CONFIG_PATH: "/usr/local/opt/imagemagick@6/lib/pkgconfig"
```

Comment:
This seems odd to me – shouldn't it be on the `$PATH`?

Reply:
I only copy/pasted that from the `test` job tbh. I think that's a special env var that get checked by `pkg_config` – the took used behind the scenes when brew tries to build imagemagick during install.

Not even sure why we'd need ImageMagick there, but I figured that should be a mystery to solve for another day 😛